src/LinuxTracing/TracerImpl.h

```diff
@@ -53,6 +55,9 @@ class TracerImpl : public Tracer {
   void Start() override;
   void Stop() override;
 
+  void ProcessFunctionEntry(const orbit_grpc_protos::FunctionEntry& function_entry) override;
```

Comment:
I guess I would add a comment (similar to the one in ProducerEventProcessorHijackingFunctionEntryExitForLinuxTracing), briefly explaining why those methods are here (basically the hijacking).

Reply:
Done, but in the interface rather than here.